BRAD/agent.py

```
@@ -989,22 +995,46 @@ class AgentFactory():
         if self.session_path:
             agent = Agent(interactive=self.interactive, tools=self.tool_modules, restart=self.session_path)
+        elif self.start_path:
+            agent = Agent(interactive=self.interactive, tools=self.tool_modules, start_path=self.start_path)
         else:
             agent = Agent(interactive=self.interactive, tools=self.tool_modules)
 
+
+        if self.db_name and self.persist_directory:
+            db, _ = agent.load_literature_db(persist_directory=self.persist_directory, db_name=self.db_name)
+            agent.state['databases']['RAG'] = db
+        elif self.db_name == None and self.persist_directory==None:
```

Ruff

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)